webapp/src/main/webapp/news-details/components/ExoNewsDetails.vue

```diff
@@ -380,23 +171,6 @@ export default {
         }
       }, redirectionTime);
     },
-    targetBlank: function (content) {
-      const internal = location.host + eXo.env.portal.context;
-      const domParser = new DOMParser();
-      const docElement = domParser.parseFromString(content, 'text/html').documentElement;
-      const links = docElement.getElementsByTagName('a');
-      for (let i=0; i < links.length ; i++) {
-        let href = links[i].href.replace(/(^\w+:|^)\/\//, '');
-        if (href.endsWith('/')) {
-          href = href.slice(0, -1);
-        }
-        if (href !== location.host && !href.startsWith(internal)) {
-          links[i].setAttribute('target', '_blank');
-          links[i].setAttribute('rel', 'noopener noreferrer');
-        }
-      }
-      return docElement.innerHTML;
-    },
     postNews(schedulePostDate, postArticleMode, publish) {
       this.news.timeZoneId = USER_TIMEZONE_ID;
       this.news.pinned = publish;
```

Comment:
```suggestion
      this.news.published= publish;
```

Reply:
In the back end the name of field is pinned.